packages/babel-core/src/config/files/configuration.js

```diff
@@ -230,7 +231,10 @@ const readConfigJSON5 = makeStaticFileCache((filepath, content) => {
   try {
     options = json5.parse(content);
   } catch (err) {
-    err.message = `${filepath}: Error while parsing config - ${err.message}`;
+    const code = codeFrame(content, err.lineNumber, err.columnNumber);
```

Comment:
We should check typeof err.lineNumber === "number" when generating the code frame and skip it otherwise, I think, since .lineNumber isn't a speced property.